### PR TITLE
 Advice user does command ssh on non existing profile 

### PR DESCRIPTION
**Pull request by RA489**
When minikube profile is delete and try to do minkube ssh , it crashes with error
fixes https://github.com/kubernetes/minikube/issues/4889

### CODE REVIEW

File: pkg/minikube/problem/err_map.go [hunk 1]
```diff
@@ -30,6 +30,11 @@ var vmProblems = map[string]match{
 		Advice: "Please make sure the service you are looking for is deployed or is in the correct namespace.",
 		Issues: []int{4599},
 	},
+	"PROFILE_NOT_FOUND": {
+		Regexp: re(`Could not find the minikube profile`),
```
Comment (medyagh) [inline pkg/minikube/problem/err_map.go]: the regex should be,  "Machine does not exist for api.Exists(lis)"

File: pkg/minikube/problem/err_map.go [hunk 2]
```diff
@@ -30,6 +30,11 @@ var vmProblems = map[string]match{
 		Advice: "Please make sure the service you are looking for is deployed or is in the correct namespace.",
 		Issues: []int{4599},
 	},
+	"PROFILE_NOT_FOUND": {
```
Comment (medyagh) [inline pkg/minikube/problem/err_map.go]: this should be "MACHINE_NOT_FOUND"

File: pkg/minikube/problem/err_map.go (hunk 1 above)
Comment (RA489) [reply pkg/minikube/problem/err_map.go]: Done!

File: pkg/minikube/problem/err_map.go (hunk 2 above)
Comment (RA489) [reply pkg/minikube/problem/err_map.go]: Done!

**Comment by RA489** [comment]
@medyagh Thanks for your reviews! I have updated the PR as per your suggestions.

File: pkg/minikube/problem/err_map.go [hunk 3]
```diff
@@ -30,6 +30,11 @@ var vmProblems = map[string]match{
 		Advice: "Please make sure the service you are looking for is deployed or is in the correct namespace.",
 		Issues: []int{4599},
 	},
+	"MACHINE_NOT_FOUND": {
+		Regexp: re(`Machine does not exist for api.Exists(lis)`),
```
Comment (medyagh) [inline pkg/minikube/problem/err_map.go]: the (lis) is extra ( that was my example profile)
it should be "Machine does not exist for api.Exists"

I recommend attaching the output of the console to try, I recmmend
deleting your minikube then 
then try ./out/minikube ssh -p p1 -- ls -lah

then paste the output in the PR review so we know it worked :)

Comment (RA489) [reply pkg/minikube/problem/err_map.go]: sure

Comment (RA489) [reply pkg/minikube/problem/err_map.go]: Done!

**Comment by RA489** [comment]
@medyagh attaching the output of the console ```
root@openstack-nti-18:/home/ubuntu/k8sworkspaces/minikube/out# minikube ssh -p p1 -- ls -lah

💣  Error getting host: Machine does not exist for api.Exists(p1)

😿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
👉  https://github.com/kubernetes/minikube/issues/new

```

**Comment by tstromberg** [comment]
I don't think this is taking the optimal approach: the error is still opaque.

Rather than advice, I would prefer that ssh to exit with a very clear message, such as:

`There is no active "minikube" cluster to ssh to.`

I took a poke around, and think this could be solved by updating ssh.go to use `api.Exists`, such as:

```
	exists, err := api.Exists(machineName)
	if err != nil {
		return exit.WIthError(...)
	}
	if !exists {
               return exit.WithCode(exit.Data, "crafty error message")
       }
```

Alternatively, I saw that `dashboard.go` does some fancy error type checking, but `api.Exists` seems the clearest.

**Comment by medyagh** [comment]
@RA489 Thank you for this contribution ! this will make user experience better

**Pull request merged**